src/components/Heading/Heading.component.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+//import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import styled from 'styled-components';
+
+const HeadingH1 = styled.h1`
```

Comment:
We could use a meaningful name for the styled-component because that's the main idea of the styled-components, for example `title`

Reply:
That makes sense. I have just changed it to `title`.